R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -196,6 +196,12 @@ test_that("create DataFrame from RDD", {
   expect_equal(dtypes(df), list(c("name", "string"), c("age", "int"), c("height", "float")))
   expect_equal(as.list(collect(where(df, df$name == "John"))),
                list(name = "John", age = 19L, height = 176.5))
+  expect_equal(getNumPartitions(toRDD(df)), 1)
```

Comment:
hmm, I'm not finding any automatic assignment to numSlices in Python or ParallelCollectionRDD ([here](https://github.com/apache/spark/blob/39e2bad6a866d27c3ca594d15e574a1da3ee84cc/core/src/main/scala/org/apache/spark/rdd/ParallelCollectionRDD.scala#L118))

I see [defaultParallelism ](https://github.com/apache/spark/blob/31da755c80aed8219c368fd18c72b42e50be46fc/core/src/main/scala/org/apache/spark/SparkContext.scala#L2272). I thought about be smarter than fixing the default partition number at 1, but
a) it's been this way since Spark 1.3
b) most R data size loaded from the driver is small anyway and might not be worthwhile to have more than 1 partition